src/openforms/submissions/tests/form_logic/test_submission_logic.py

```diff
@@ -907,6 +907,106 @@ def test_component_values_hidden_fieldset_used_in_subsequent_logic(self):
+                {
+                    "variable": "datetime",
+                    "action": {
+                        "type": "variable",
+                        "value": {"var": "now"},
+                    },
+                }
+            ],
+        )
+        submission = SubmissionFactory.create(form=form)
+        self._add_submission_to_session(submission)
+        logic_check_endpoint = reverse(
+            "api:submission-steps-logic-check",
+            kwargs={
+                "submission_uuid": submission.uuid,
+                "step_uuid": form.formstep_set.get().uuid,
+            },
+        )
+
+        with freeze_time("2024-03-18T08:31:08+01:00"):
+            response = self.client.post(logic_check_endpoint, {"data": {}})
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+        new_value = response.json()["step"]["data"]["datetime"]
+        # check that the seconds/ms are truncated to prevent infinite logic bouncing.
+        # Note that this doesn't make the problem go away 100% - you will get an
+        # additional check if the minute value changes, but that should settle after one
+        # extra logic check.
+        self.assertEqual(new_value, "2024-03-18T07:31:00Z")
+
+    def test_component_value_set_to_today(self):
+        form = FormFactory.create(
+            generate_minimal_setup=True,
+            formstep__form_definition__configuration={
+                "components": [
+                    {
+                        "key": "date",
+                        "type": "date",
+                        "label": "Today",
+                    },
+                ]
+            },
+        )
+        FormLogicFactory.create(
+            form=form,
+            json_logic_trigger=True,
+            actions=[
+                {
+                    "variable": "date",
+                    "action": {
+                        "type": "variable",
+                        "value": {"var": "today"},
+                    },
+                }
+            ],
+        )
+        submission = SubmissionFactory.create(form=form)
+        self._add_submission_to_session(submission)
+        logic_check_endpoint = reverse(
+            "api:submission-steps-logic-check",
+            kwargs={
+                "submission_uuid": submission.uuid,
+                "step_uuid": form.formstep_set.get().uuid,
+            },
+        )
+
+        with freeze_time("2024-03-18T08:31:08+01:00"):
+            response = self.client.post(logic_check_endpoint, {"data": {}})
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+        new_value = response.json()["step"]["data"]["date"]
+        # check that the seconds/ms are truncated to prevent infinite logic bouncing.
+        # Note that this doesn't make the problem go away 100% - you will get an
+        # additional check if the minute value changes, but that should settle after one
+        # extra logic check.
```

Comment:
I don't think this comment is relevant for the `today` variable?

Reply:
correct, I'll update!